### PR TITLE
first draft of the testing guide

### PR DESCRIPTION
**Pull request by jdob**

### CODE REVIEW

File: docs/testing-guide.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# Testing Built Images
+
+Images may be run on either bare metal or virtualization. The following guide provides examples to speed
```
Comment (dbw7) [inline docs/testing-guide.md:3]: I think "or through virtualization" may flow a bit better

File: docs/testing-guide.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+![images](./images/libvirt-iso-4.png)
+
+Name the VM and select Finish:
+
+![images](./images/libvirt-iso-5.png)
+
+After pressing "Finish", the VM will boot. Depending on the version of the SLE Micro self-install ISO, it may
+prompt you to manually select the "Install SLE Micro" option before proceeding. In either case, stop the installation
+by using the "Shutdown -> Force Off" option under the "Virtual Machine" menu. See the next step for more information
+of what needs to happen before the installation can continue.
```
Comment (dbw7) [inline docs/testing-guide.md:46]: I think "information of" should be "information on" or "information about"

File: docs/testing-guide.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+says "No media selected", use the "Browse" button to navigate to your ISO built by EIB. Once it is selected, press
+"Apply".
+
+![images](./images/libvirt-iso-7.png)
+
+Select the "Boot Options" entry on the left. Ensure that the "CDROM" entry is checked and press "Apply".
+
+![images](./images/libvirt-iso-8.png)
+
+Once those two configuration value have been set, navigate back to the "Console" option under "View" and power on
```
Comment (dbw7) [inline docs/testing-guide.md:60]: Small error, "values" instead of "value"

**Review by dbw7 — COMMENTED** [review]
The guide is great and very clear. Was able to use it to get the elemental configuration of combustion working and everything. Only a few small nits that I've commented on

File: README.md [hunk 4]
```diff
@@ -12,31 +12,88 @@ podman build -t eib:dev .
...
+├── eib-config-iso.yaml
+├── eib-config-raw.yaml
 └── images
     └── SLE-Micro.x86_64-5.5.0-Default-SelfInstall-GM.install.iso
+    └── SLE-Micro.x86_64-5.5.0-Default-GM.raw
 ```
 
-That directory must be attached to the container at runtime. The following command attaches the directory and runs
-EIB against the volume (replace `$IMAGE_DIR` with your local configuration directory):
+* `eib-config.yaml` - All image configuration files should be in the root of the image configuration directory.
```
Comment (atanasdinov) [inline README.md:62]:
```suggestion
* `eib-config-iso.yaml`, `eib-config-raw.yaml` - All image configuration files should be in the root of the image configuration directory.
```
The specified file no longer exists in the new dir structure.

**Review by atanasdinov — COMMENTED** [review]
It looks great, good job!

File: docs/testing-guide.md (hunk 2 above)
Comment (jdob) [reply docs/testing-guide.md:46]: Yup, this should be "on". Good eye.

File: README.md (hunk 4 above)
Comment (jdob) [reply README.md:62]: Good eye, ty.

**Review by dbw7 — APPROVED** [review]
Looks good to me!

**Pull request merged**